streams/upgrade-system-tests-31/src/test/java/org/apache/kafka/streams/tests/StreamsUpgradeTest.java

```diff
@@ -63,21 +80,30 @@ public static void main(final String[] args) throws Exception {
         }));
     }
 
-    private static <KIn, VIn, KOut, VOut> ProcessorSupplier<KIn, VIn, KOut, VOut> printProcessorSupplier() {
+    private static void buildFKTable(final KStream<String, Integer> primaryTable,
+        final KTable<Integer, String> otherTable) {
```

Comment:
nit: indention

Reply:
Done in [d530a29](https://github.com/apache/kafka/pull/12122/commits/d530a29d1df6bb5302745c54ef08c83dc6c9c6de).